src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabController.js

```diff
@@ -11,6 +11,30 @@
      */
     onNewBatchClick: function(component, event, helper) {
         helper.checkFieldPermissions(component);
+    },
+
+    loadMoreData: function (component, event, helper) {
+        event.getSource().set("v.isLoading", true);
+        if (component.get('v.batchData').length >= component.get('v.totalNumberOfRows')) {
+            component.set('v.enableInfiniteLoading', false);
+            event.getSource().set("v.isLoading", false);
```

Comment:
Should we / can we be consistent with usage of single quotes vs double quotes on .set("v.isLoading"), .set('v.isLoading') etc?